.gitignore

```diff
@@ -76,6 +76,10 @@ cpu.out
 /public/css
 /public/fonts
 /public/img/webpack
```

Comment:
Could remove the old entries.

Reply:
IAfter this PR gets approved.

Otherwise, when I switch branches, there are always a lot of inconsistent "ignored" files.